biomodels/src/test/java/org/reactome/release/ModelsTSVParserTester.java

```diff
@@ -2,68 +2,95 @@
 
 public class ModelsTSVParserTester {
 
     private final int correctPathwayCount = 3;
     private final int correctNumberPathwaysInTestFile = 745;
-    private final int correctNumberBioModelsIdsInTestFile = 153;
+    private final long correctNumberBioModelsIdsInTestFile = 153L;
+    private final String RESOURCE_DIR = Paths.get("src", "test", "resources").toFile().getAbsolutePath();
+
     @Test
-    public void emptyMapFromEmptyFileName() {
+    public void emptyFilenameReturnsEmptyMap() {
         Map<String, List<String>> pathwayToBioModelsIds =  ModelsTSVParser.parse("");
-        assertTrue("Empty file produces empty pathwayToBioModelsIds map", pathwayToBioModelsIds.isEmpty());
+        assertThat("Empty file produces empty pathwayToBioModelsIds map", pathwayToBioModelsIds.isEmpty(), is(true));
     }
 
     @Test
-    public void emptyMapFromUnknownFileName() {
+    public void unknownFileNameReturnsEmptyMap() {
+        String filepath = Paths.get(RESOURCE_DIR, "filedoesnotexist.tsv").toFile().getAbsolutePath();
         Map<String, List<String>> pathwayToBioModelsIds =  ModelsTSVParser.parse("filedoesnotexist.tsv");
-        assertTrue("Unknown file produces empty pathwayToBioModelsIds map", pathwayToBioModelsIds.isEmpty());
+        assertThat("Unknown file produces empty pathwayToBioModelsIds map", pathwayToBioModelsIds.isEmpty(), is(true));
     }
 
     @Test
-    public void emptyMapFromImproperFile() {
-        Map<String, List<String>> pathwayToBioModelsIds = ModelsTSVParser.parse("src/test/resources/improper_models2pathways.tsv");
-        assertTrue("Improperly formatted file produces empty pathwayToBioModelsIds map", pathwayToBioModelsIds.isEmpty());
+    public void improperlyFormattedFileReturnsEmptyMap() {
+        String filepath = Paths.get(RESOURCE_DIR, "improper_models2pathways.tsv").toFile().getAbsolutePath();
+        Map<String, List<String>> pathwayToBioModelsIds = ModelsTSVParser.parse(filepath);
+        assertThat("Improperly formatted file produces empty pathwayToBioModelsIds map", pathwayToBioModelsIds.isEmpty(), is(true));
     }
 
     @Test
-    public void populatedMapFromPartiallyCorrectFile() {
-        Map<String, List<String>> pathwayToBioModelsIds = ModelsTSVParser.parse("src/test/resources/partiallycorrect_models2pathways.tsv");
-        assertEquals(pathwayToBioModelsIds.keySet().size(), correctPathwayCount);
-        assertTrue(pathwayToBioModelsIds.get("R-MMU-1169091").contains("BIOMD0000000226"));
+    public void partiallyCorrectFileReturnsPopulatedMap() {
+        String filepath = Paths.get(RESOURCE_DIR, "partiallycorrect_models2pathways.tsv").toFile().getAbsolutePath();
+        Map<String, List<String>> pathwayToBioModelsIds = ModelsTSVParser.parse(filepath);
+        assertThat(pathwayToBioModelsIds.keySet().size(), is(equalTo(correctPathwayCount)));
+        assertThat(pathwayToBioModelsIds.get("R-MMU-1169091").contains("BIOMD0000000226"), is(equalTo(true)));
```

Comment:
Try to avoid testing against a literal boolean value if possible.  Hamcrest usually has matchers for the common scenarios to test like here to see if a value is contained in a list.  In this case, the assert can be written as:

`assertThat(pathwayToBioModelsIds.get("R-MMU-1169091"), contains("BIOMD0000000226"));`